src/tribol/mesh/CouplingScheme.cpp

```diff
@@ -517,29 +517,39 @@ bool CouplingScheme::isValidCase()
       this->m_contactCase = NO_CASE;
    }
 
-   // specify auto-contact specific interpenetration check and verify 
-   // element thicknesses have been registered
    parameters_t& params = parameters_t::getInstance();
-   if (this->m_contactCase == AUTO)
-   { 
-      params.auto_interpen_check = true;
+   if (this->m_contactMethod == COMMON_PLANE)
```

Comment:
What does `ContactCase::AUTO` do after this change?

Reply:
I'll test elsewhere, but in principle there is no change. What this design does preclude is specifying auto contact with a tied use case. I would argue that auto, however, is for instances when you truly do not know what is in contact with what, or the number of contact surfaces in a coupling scheme is too burdensome to individually specify. In a TIED application, you know exactly what is in contact and specifying surface boundary attributes shouldn't be a barrier to entry. Physically speaking, too, auto will most likely treat multiple mesh interfaces, especially self contact and cases where you don't know what will be in contact, but TIED specifically treats different topological entities where specifying separate boundary attributes should be easy enough to do.

Reply:
Makes sense.  We might want to document this, perhaps in `Parameters.hpp`?

Reply:
Added to the doxygen comment for the ContactCase.